module/renderer/src/androidUnitTest/kotlin/org/cru/godtools/shared/renderer/content/RenderSpacerPaparazziTest.kt

```diff
@@ -0,0 +1,138 @@
+package org.cru.godtools.shared.renderer.content
+
+import com.android.ide.common.rendering.api.SessionParams.RenderingMode
+import kotlin.test.Test
+import org.cru.godtools.shared.renderer.BasePaparazziTest
+import org.cru.godtools.shared.tool.parser.model.Spacer
+import org.cru.godtools.shared.tool.parser.model.Text
+
+class RenderSpacerPaparazziTest : BasePaparazziTest(renderingMode = RenderingMode.SHRINK) {
+
+    @Test
+    fun `RenderSpacer() - Auto Spacer On Top`() = contentSnapshot {
+
```

Comment:
the blank line at the start of code blocks like this is causing ktlint code style violations.

you can run `./gradlew ktlintCheck` or `./gradlew ktlintFormat` to check/auto-format the code